index.js

```diff
@@ -0,0 +1,3 @@
+const nombreUsuario = prompt('Ingrese su nombre ');
+alert('Hola Usuario!');
```

Comment:
En este caso creo que podriamos realizar un cambio

```suggestion
alert('Hola' + nombreUsuario);
```